Tests/MoyaProviderSpec.swift

```diff
@@ -794,7 +796,85 @@ final class MoyaProviderSpec: QuickSpec {
+                    return OHHTTPStubsResponse(data: GitHubUserContent.downloadMoyaWebContent("logo_github.png").sampleData, statusCode: 200, headers: ["Content-Length": ""]).responseTime(-4)
+                })
+                provider = MoyaProvider<GitHubUserContent>()
+            }
+
+            it("tracks progress of download request") {
+                let target: GitHubUserContent = .downloadMoyaWebContent("logo_github.png")
+
+                var progressObjects: [Progress?] = []
+                var progressValues: [Double] = []
+                var completedValues: [Bool] = []
+                var error: MoyaError?
+
+                waitUntil(timeout: 5.0) { done in
+                    let progressClosure: ProgressBlock = { progress in
+                        progressObjects.append(progress.progressObject)
+                        progressValues.append(progress.progress)
+                        completedValues.append(progress.completed)
+                    }
+
+                    let progressCompletionClosure: Completion = { (result) in
+                        if case .failure(let err) = result {
+                            error = err
+                        }
+                        done()
+                    }
+
+                    provider.request(target, callbackQueue: nil, progress: progressClosure, completion: progressCompletionClosure)
+                }
+
+                expect(error).to(beNil())
+                expect(progressValues) == [0, 0, 0, 0, 1.0]
+                expect(completedValues) == [false, false, false, false, true]
+                expect(progressObjects.filter { $0 != nil }.count) == 5
+            }
+
+            it("tracks progress of a request") {
+                let target: GitHubUserContent = .requestMoyaWebContent("logo_github.png")
+
+                var progressObjects: [Progress?] = []
+                var progressValues: [Double] = []
+                var completedValues: [Bool] = []
+                var error: MoyaError?
+
+                waitUntil(timeout: 5.0) { done in
+                    let progressClosure: ProgressBlock = { progress in
+                        progressObjects.append(progress.progressObject)
+                        progressValues.append(progress.progress)
+                        completedValues.append(progress.completed)
+                    }
+
+                    let progressCompletionClosure: Completion = { (result) in
+                        if case .failure(let err) = result {
+                            error = err
+                        }
+                        done()
+                    }
+
+                    provider.request(target, callbackQueue: nil, progress: progressClosure, completion: progressCompletionClosure)
+                }
+
+                expect(error).to(beNil())
+                expect(progressValues) == [0, 0, 0, 0, 1.0]
```

Comment:
This is just curiosity, there is a reason for all moya equal tests use `expect(value) ==` instead of `.to(equal(value))`?

Reply:
@LucianoPAlmeida I guess I just use it because it's a lot more convenient to just type `==` instead of `.to(equal(value))`.